src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/util/ElinksReferenceDataClient.java

```diff
@@ -347,11 +180,10 @@ public String getAndReturnBearerToken(String userId, String role) {
         return bearerToken;
     }
 
-    public String setAndReturnJwtToken() {
-        if (StringUtils.isBlank(JWT_TOKEN)) {
-            JWT_TOKEN = generateS2SToken("rd_judicial_api");
+    public void setAndReturnJwtToken() {
```

Comment:
minor: method does not return anything anymore, possibly rename method to 'setJwtToken'

Reply:
refactored the code and moved this code block to calling method.